main.js

```diff
@@ -0,0 +1,85 @@
+
+
+const render = new Renderer()
+let board = new GoldRush()
+
+Handlebars.registerHelper('ifEquals', function (arg1, arg2, options) {
+    return (arg1 == arg2) ? options.fn(this) : options.inverse(this);
+});
+
+board.loadBoard(10, 10)
+render.renderBoard(board.getMatrix(), board.getPoints())
+$("body").keypress(function (e) {
```

Comment:
When declaring a new variable you should declare it with _const_ or _let_, otherwise, the variable will be defined as "_var_"